shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt

```diff
@@ -69,7 +69,7 @@ interface ScreenshotTest {
         heightInPx: Int? = null,
         widthInPx: Int? = null,
         name: String? = null
-    ) = compareScreenshot(view = fragment.view!!, heightInPx = heightInPx, widthInPx = widthInPx, name = name)
+    ) = compareScreenshot(view = fragment.requireView(), heightInPx = heightInPx, widthInPx = widthInPx, name = name)
```

Comment:
Corrected Lint warning: `requireView()` exception message is more specific than `NullPointerException`.

https://developer.android.com/reference/kotlin/androidx/fragment/app/Fragment#requireview